guides/common/modules/proc_assigning-ansible-roles-to-an-existing-host.adoc

```diff
@@ -0,0 +1,22 @@
+[id="adding-ansible-roles-to-an-existing-host_{context}"]
+= Assigning Ansible Roles to an Existing Host
+
+You can use Ansible roles for remote management of {RHEL} versions 8, 7, and 6.9 or later.
+
+.Prerequisites
+
+* You must configure your deployment to run Ansible roles. Add xref here.
```

Comment:
I think this might be also one to add a hard link to after this PR has merged. I've rewritten to try and make it is obvious, without breaking FCC's Every Page Is Page One idea.